server/container/util-java8/src/test/java/org/apache/james/util/streams/FakeSmtp.java

```diff
@@ -0,0 +1,54 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.util.streams;
+
+import java.net.Socket;
+
+import javax.net.SocketFactory;
+
+import com.jayway.awaitility.Awaitility;
+import com.jayway.awaitility.Duration;
+import com.jayway.awaitility.core.ConditionFactory;
+
+public class FakeSmtp extends SwarmGenericContainer {
+    public FakeSmtp() {
+        super("weave/rest-smtp-sink:latest");
+    }
+
+    public static void await(SwarmGenericContainer container) {
```

Comment:
it's sooooo sad. Why don't you want to contribute to testcontainers ?

Reply:
Takes time? Of course, I'd be happy to do it, but we also need to wait the potential contribution to be merged upstream and end up with a a temporary solution anyway.

Reply:
Nope, the wait strategy is a class, you can have it in your project and propose it for integration in testcontainers in //.